### ISSUE TITLE
request and bug report...

### ISSUE THREAD

**Issue by olearydj** [tdaykin/obsidian_image_collector#2]
Hey, thanks for sharing this tool.

I wanted to point out that it some of us use markdown image embeds that look something like `![[image.png|275]]` where the pipe symbol separates the image information from metadata used by some themes, etc to control the display of the image. In this case 275 specifies the display size. Your plugin doesn't handle these, but could easily do so by simply stripping any `|*` it finds. I may be oversimplifying it, but you get the idea.

Also, it would be nice if it would allow us to specify where the resulting images are collected to.

I have been just putting all my attachments in a single folder but that approach is not a good long term solution. I'd like to be able to use your plugin to move all the images associated with a file into a subdirectory of the folder that holds it, rather than a directory named based on the filename in the root of the vault. A setting plugin setting would allow for that sort of thing. Alternatively you could draft off the current obsidian setting for "default location for new attachments." I've changed that to use the attachments subfolder for all future attachments, so you could default to that same approach.

Anyway, a bit wordy and take it or leave it, but thanks again for sharing and good luck finishing up the degree. You have a nice portfolio and are well-positioned to land a tech gig as you hope.

**Comment by tdaykin** [comment]
Thanks for pointing out the pipe issue. I actually also just discovered it last night when I was playing around with that type of formatting to reduce image size, and yes, the plugin stopped working. I'll try and get on that ASAP. As for the subdirectory, I will spend some time thinking about it and maybe offer the option in settings to keep the plugin as is, then as you mentioned, offer the user to provide an alternative path to store images collected from notes.

Also, thank you for checking out my portfolio, and I am glad you enjoyed it; your comments mean a lot!

**Comment by olearydj** [comment]
Sounds good, thank you!

I always look at portfolios to see what interesting stuff folks are up to. In part because I use to hire a lot and in part because now I always tell my students to make them. Folks that treat college as an excuse to build a portfolio (rather than just get grades) are the ones that win. Nothing beats building things.

**Comment by tdaykin** [comment]
Yes, I agree with having a portfolio, it really helps collect things that I have done (even though I am way behind on the website). In my most recent interview for internships, I was describing a robotics project and I directed the interviewers to my portfolio and they got to see gifs, pictures, and diagrams of the project, all while I was describing details to them. It worked out very nice! 

I have also updated the plugin to exclude pipes; it seems to be working for me with the tests that I have done. Let me know if it works for you, I would appreciate that! (To update just go into Obsidian and find the plugin and click update)

Lastly, I am unsure of what is common to do with Issues, but since the bug appears to be fixed I will close this Issue soon and keep in mind the subdirectory idea. That won't be as simple as a fix, so I probably will not be able to get to it until early April.

Again, thank you for your support!
